kmir/k-src/mir-configuration.md

```diff
@@ -124,7 +124,7 @@ We represent `LocalDecls` as a cell `Map` of multiplicity `"*"`. The fields of `
 ### [LocalInfo](https://github.com/rust-lang/rust/blob/bda32a4023b1d3f96e56e1b2fc7510324f430316/compiler/rustc_middle/src/mir/mod.rs#LL887-L905C2)
 
 ```k
-  syntax DefId ::= Int // TODO: what's that? https://github.com/rust-lang/rust/blob/bda32a4023b1d3f96e56e1b2fc7510324f430316/src/tools/rust-analyzer/crates/hir-ty/src/interner.rs#L63
+  syntax DefId ::= Int
```

Comment:
It is nor clear what is the meaning of `DefId` and what is `InternId`, and the type of the latter.
https://github.com/rust-lang/rust/blob/bda32a4023b1d3f96e56e1b2fc7510324f430316/src/tools/rust-analyzer/crates/hir-ty/src/interner.rs#L63
The link provided doesnot lead to any meaning explanation of the `InternId`.
However, `DefId` does appear in unoptimised Mir output of rust programs. 
I've created issue #134 to track this problem.

Furthermore, `LocalInfo` seems to be information collacted for static analysis, a.k.a., rust analyzer. It might not be a part of the execution semantics. Shall we comment it out?

Reply:
The `DefId` type seems to be coming for HIR (see the [docs](https://rustc-dev-guide.rust-lang.org/hir.html)) and identify a definition in a crate. It is not clear to me how (and what for) to use it.